src/components/Global/Navigation/Navigation.tsx

```diff
@@ -1,13 +1,25 @@
 /* External dependencies */
 import React, { useEffect, useState, useRef } from 'react';
-import { NavLink, useLocation } from 'react-router-dom';
+import { useLocation, NavLink } from 'react-router-dom';
 import classNames from 'classnames/bind';
 
 /* Internal dependencies */
 import styles from './Navigation.module.scss';
 
 const cx = classNames.bind(styles);
 
+function ExNavLink({ to, activePath, children }) {
+  const location = useLocation();
+  const active = activePath.includes(location.pathname);
+  let style = {};
+  if (active) style = { color: '#2ac1bc' };
+  return (
+    <NavLink to={to} style={style}>
+      {children}
+    </NavLink>
+  );
```

Comment:
```suggestion
  const isActive = useCallback((match, location) => {
    return activePath.includes(location.pathname);
  }, [activePath])

  return (
    <NavLink to={to} isActive={isActive} activeClassName={className}>
      {children}
    </NavLink>
  );
```
이미 NavLink컴포넌트에서는 비슷한 기능을 제공하고 있어서 위처럼 사용해봐도 좋을것 같습니다. `activeClassName`는 sass를 통해서 className를 지정해주어도 되고 만약 `activeStyle`을 사용한다면 해당 스타일은 컴포넌트 바깥에
```
const ACTIVE_STYLE = {
  ...
}
```
와 같이정의해주는게 메모리 측면에서 좋을것 같습니다

Reply:
감사합니다! 훨씬 깖끔하네요